networking/libsnnet/ovs_bridge.go

```diff
@@ -34,3 +34,10 @@ func vsctlCmd(args []string) error {
 
 	return nil
 }
+
+func ovsGetDevice(bridgeId string) error {
```

Comment:
Since this is just a wrapper for createOvsBridge, let's either change this or remove it.

1. change it to query ovs-vsctl to search for an existing bridge with that name
2. just call createOvsBridge instead.

Now that I'm thinking about it, option 1 seems better.